src/controllers/user/createUser.ts

```diff
@@ -69,7 +81,7 @@ export const createUser = async (req: Request, res: Response) => {
 
   } catch (err) {
     console.log("Error creating user:", err.message)
-    if (err.message .substring(0,46) == "duplicate key value violates unique constraint") {
+    if (err.message.substring(0,46) == "duplicate key value violates unique constraint") {
       return res.status(400).json({ message: "Email or Phone Number already exists" })
```

Comment:
Also keep separate errors for phone numbers and emails. Since you are doing so much, might as well go the whole distance lol

Reply:
So I asked Soumitra this on the server and he said this is how we should do it.

Reply:
Sorry wrong reply. I'll fix this